_source/_posts/2020-07-02-spring-boot-using-java-modules.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: blog_post
+title: "Build a Spring Boot application using Java 9+ modules"
+author: bruno-leite
+by: contractor
+communities: [java]
+description: "This tutorial shows how to create a multi-module Spring Boot application using Java 9+ modular system"
+tags: [spring-boot, java, modules]
+tweets:
+- "Learn how to create a @springboot application using Java 9+ modular system"
+- "Tutorial: Java 9+ modules with @springboot"
+- "Spring Boot + Java Modules? Check it out"
+image:
+type: conversion
+---
+
+Java is one of the most mature and persistent development languages that exist. Recently it entered into a 6-month release schedule which enabled to deliver more frequent updates to the language.
+One of that changes was the modular system that is available since Java 9.
```

Comment:
```suggestion
One of those changes was the modular system that is available since Java 9.
```

If you don't have one already, I'd suggest getting an account on grammarly.com.
I usually cut/paste my posts in there and it finds a lot of minor errors. It doesn't handle markdown, so it will show errors with backticks, but I just ignore those ones.